src/main/java/dk/sdu/se_f22/brandmodule/index/BrandIndex.java

```diff
@@ -17,7 +21,34 @@
 
     @Override
     public void indexBrandInformation(Brand brand, List<String> tokens) {
-
+        try {
+            PreparedStatement mapInsert = DBConn.prepareStatement("INSERT INTO tokenbrandmap(brandid, tokenid) VALUES (?,?)");
+            PreparedStatement tokenInsert = DBConn.prepareStatement("INSERT INTO tokens(token) VALUES (?)");
+            PreparedStatement queryToken = DBConn.prepareStatement("SELECT token FROM tokens");
+
+            ResultSet rs = queryToken.executeQuery();
+
+            ArrayList<String> newTokens = new ArrayList<>(tokens);
+
+            for (int i = 0; i < tokens.size(); i++){
+                while (rs.next()){
+                    if (rs.getString("token") != newTokens.get(i)){
```

Comment:
"token" seems to be a magic String.

Reply:
Made a change to the code, made a variable for "token" with a more appropriate name, we don't quite understand what you mean.